plasma_framework/contracts/src/framework/BlockController.sol

```diff
@@ -0,0 +1,76 @@
+pragma solidity ^0.5.0;
+
+import "./models/BlockModel.sol";
+import "./registries/VaultRegistry.sol";
+import "./utils/Operated.sol";
+
+contract BlockController is Operated, VaultRegistry {
+    uint256 private _childBlockInterval;
+    uint256 private _nextChildBlock;
+    uint256 private _nextDepositBlock;
+
+    mapping (uint256 => BlockModel.Block) private _blocks;
+
+    event BlockSubmitted(
+        uint256 blockNumber
+    );
+
+    constructor(uint256 _interval) public {
+        _childBlockInterval = _interval;
+        _nextChildBlock = _childBlockInterval;
+        _nextDepositBlock = 1;
+    }
+
+    function blocks(uint256 _blockNumber) public view returns (bytes32 root, uint256 timestamp) {
```

Comment:
If you kept `_blocks` public, this getter would not be necessary.

Reply:
answered up in previous comment, would like to hear your opinion.